pipelines/build-testapp.yml

```diff
@@ -94,9 +94,256 @@ stages:
         targetPath: $(artifactDir)
         artifactName: $(artifactName)
 
+  - job: test_app_android
+    timeoutInMinutes: 180
+    displayName: 'test app Android'
+
+    pool:
+      vmImage: 'windows-2022'
+
+    variables:
+    - group: model-perf-variables
+      
+    steps:
+    - checkout: self
+      lfs: true
+    
+    - task: JavaToolInstaller@0
+      displayName: 'use Java 17'
+      inputs:
+        versionSpec: '17'
+        jdkArchitectureOption: 'x64'
+        jdkSourceOption: 'PreInstalled'
+
+    - task: PowerShell@2
+      displayName: 'build test apps for Android'
```

Comment:
'b'->'B', and maybe 'Build Android Test App' is better?

Reply:
updated